docs/content/testing/testing-overview.md

```diff
@@ -0,0 +1,43 @@
+**Warning:** The [pre-alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release is for evaluation purposes only, with limited documentation - see the guidance on [Recommended use](../../../README.md#recommended-use).
+
+-----
+
+# Testing overview
+
+We use testing to validate functionality and to ensure that the parts of the Unity GDK and the framework as a whole is resilient under different conditions.
+
+## Test categories
+
+We have three main test categories:
+- Tools tests
+- Code Generator tests
+- Unity tests
+
+All three of these categories are built using the NUnit testing framework.
+
+## Where to find tests code
+
+The documentation linter tests can be found in `tools\DocsLinter\Tests`.
```

Comment:
If you start these lines with:
 Tools tests...
Code Generator tests...
Unity tests...

it's more obvious that you're following the order above.